src/test/erasure-code/CMakeLists.txt

```diff
@@ -83,9 +83,12 @@ target_link_libraries(unittest_erasure_code_plugin_jerasure
   global
   osd
   ec_jerasure
-  common)
+  common
+  ${UNITTEST_LIBS}
+  )
 add_dependencies(unittest_erasure_code_plugin_jerasure
   ec_jerasure)
+set_target_properties(unittest_erasure_code_plugin_jerasure PROPERTIES COMPILE_FLAGS ${UNITTEST_CXX_FLAGS})
```

Comment:
`add_ceph_unittest(unittest_erasure_code_plugin_jerasure ...)` takes care of this for us. could you print out the `${UNITTEST_LIBS}` and `${UNITTEST_CXX_FLAGS}` in `add_ceph_unittest()`? to check if these variables are as expected?